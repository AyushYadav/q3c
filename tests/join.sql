select count(*) from test1 as a, test as b where q3c_join(a.ra,a.dec,b.ra,b.dec,0.002);
select count(*) from test1 as a, test as b where q3c_join(a.ra,a.dec,b.ra,b.dec,0.004);
select count(*) from test1 as a, test as b where q3c_join(a.ra,a.dec,b.ra,b.dec,0.01);
select count(*) from test1 as a, test as b where q3c_join(a.ra,a.dec,b.ra,b.dec,0.02);
select count(*) from test1 as a, test as b where q3c_join(a.ra,a.dec,b.ra,b.dec,0.04);
