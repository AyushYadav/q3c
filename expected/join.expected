 count 
-------
  1343
(1 row)

 count 
-------
  1462
(1 row)

 count 
-------
  7768
(1 row)

 count 
-------
 28035
(1 row)

 count  
--------
 137000
(1 row)

